### ISSUE TITLE
The examples listed here do not work

### ISSUE THREAD

**Issue by vivere-dally** [microsoftgraph/microsoft-graph-docs#14323]
In my scenario, I need to use the [Client credentials provider](https://docs.microsoft.com/en-us/graph/sdks/choose-authentication-providers?tabs=CS#client-credentials-provider), but the configuration is different from the one listed.

Using the `var scopes = new[] { "User.Read.All" };` will cause an error:
> AADSTS1002012: The provided value for scope User.Read.All is not valid. Client credential flows must have a scope value with /.default suffixed to the resource identifier (application ID URI.

This error message suggest that we need to use `https://graph.microsoft.com/.default` but acquiring a token with this scope will cause unauthorized errors when trying to use the `GraphServiceClient`.

How can I acquire a token with the `User.Read.All` scope? The example here is wrong.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4537f77e-a8e8-e3f1-9cc4-2fd930db7ada
* Version Independent ID: 46396fa2-f7e2-322e-5e76-f9e2d17d1b48
* Content: [Choose a Microsoft Graph authentication provider - Microsoft Graph](https://docs.microsoft.com/en-us/graph/sdks/choose-authentication-providers?tabs=CS)
* Content Source: [concepts/sdks/choose-authentication-providers.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/concepts/sdks/choose-authentication-providers.md)
* Product: **non-product-specific**
* Technology: **microsoft-graph**
* GitHub Login: @MIchaelMainer
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by jasonjoh** [comment]
@vivere-dally yes you're correct, it should be `https://graph.microsoft.com/.default`. For that scope to work, you have to add the desired permissions (in this case, `User.Read.All`) in the **API permissions** section of your app registration in the Azure portal, **AND** an admin has to grant consent.

![image](https://user-images.githubusercontent.com/8966342/134967447-b004e5d6-b0c8-4210-a295-13e42a8c63c5.png)
